manim/mobject/polyhedra.py

```diff
@@ -0,0 +1,360 @@
+                obj = Octahedron()
+                self.add(obj)
+    """
+
+    def __init__(self, edge_length: float = 1, **kwargs):
+        unit = edge_length * np.sqrt(2) / 2
+        Polyhedron.__init__(
+            self,
+            vertex_coords=[
+                np.array([unit, 0, 0]),
+                np.array([-unit, 0, 0]),
+                np.array([0, unit, 0]),
+                np.array([0, -unit, 0]),
+                np.array([0, 0, unit]),
+                np.array([0, 0, -unit]),
+            ],
+            faces_list=[
+                [2, 4, 1],
+                [0, 4, 2],
+                [4, 3, 0],
+                [1, 3, 4],
+                [3, 5, 0],
+                [1, 5, 3],
+                [2, 5, 1],
+                [0, 5, 2],
+            ],
+            **kwargs
+        )
+
+
+class Icosahedron(Polyhedron):
+    """An icosahedron, one of the five platonic solids.
+
+    Parameters
+    ----------
+    edge_length:
+        The length of an edge between any two vertices.
+
+    Examples
+    --------
+
+    .. manim:: IcosahedronScene
+        :save_last_frame:
+
+        class IcosahedronScene(ThreeDScene):
+            def construct(self):
+                self.set_camera_orientation(phi=75 * DEGREES, theta=30 * DEGREES)
+                obj = Icosahedron()
+                self.add(obj)
+    """
+
+    def __init__(self, edge_length: float = 1, **kwargs):
+        unit_a = edge_length * ((1 + np.sqrt(5)) / 4)
+        unit_b = edge_length * (1 / 2)
+        Polyhedron.__init__(
+            self,
+            vertex_coords=[
+                np.array([0, unit_b, unit_a]),
+                np.array([0, -unit_b, unit_a]),
+                np.array([0, unit_b, -unit_a]),
+                np.array([0, -unit_b, -unit_a]),
+                np.array([unit_b, unit_a, 0]),
+                np.array([unit_b, -unit_a, 0]),
+                np.array([-unit_b, unit_a, 0]),
+                np.array([-unit_b, -unit_a, 0]),
+                np.array([unit_a, 0, unit_b]),
+                np.array([unit_a, 0, -unit_b]),
+                np.array([-unit_a, 0, unit_b]),
+                np.array([-unit_a, 0, -unit_b]),
+            ],
+            faces_list=[
+                [1, 8, 0],
+                [1, 5, 7],
+                [8, 5, 1],
+                [7, 3, 5],
+                [5, 9, 3],
+                [8, 9, 5],
+                [3, 2, 9],
+                [9, 4, 2],
+                [8, 4, 9],
+                [0, 4, 8],
+                [6, 4, 0],
+                [6, 2, 4],
+                [11, 2, 6],
+                [3, 11, 2],
+                [0, 6, 10],
+                [10, 1, 0],
+                [10, 7, 1],
+                [11, 7, 3],
+                [10, 11, 7],
+                [10, 11, 6],
+            ],
+            **kwargs
+        )
+
+
+class Dodecahedron(Polyhedron):
+    """A dodecahedron, one of the five platonic solids.
```

Comment:
It might be useful to note how many sides the Polyhedron has.
```suggestion
    """A dodecahedron, one of the five platonic solids, and one with twelve faces.
```
or
```suggestion
    """A dodecahedron, the polyhedron with twelve faces.
```

Reply:
Comment above.